### DISCUSSION TITLE
Inquiry Regarding Discounted and Complimentary Licenses

### DISCUSSION THREAD

**Discussion by mrzzmrzz** [izhangzhihao/intellij-rainbow-brackets#2662] [Q&A] [answered]
I am reaching out to express my appreciation for the outstanding plugin that you have developed. It has undoubtedly caught my attention due to its exceptional functionality.

I wanted to inquire if there could be options available for discounted licenses or licenses tailored to educational purposes, similar to what other premium plugins often provide.

<img width="1265" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/36831062/5f0e8f75-ccb6-4e62-95f8-c87531632005">

**Comment by izhangzhihao** [comment, marked as answer]
Initially, this plugin supported student licenses for a period of time. However, it was found that around 60% to 70% of sales came from free student licenses. Considering the abuse of student licenses by individuals who are not actual students, the provision of student licenses was suspended. Just last week, I also offered free student licenses to the public (via email application). However, JetBrains informed me the day before yesterday that offering student licenses externally is no longer permissible. Therefore, there won't be student licenses available in the future. If I were to offer student licenses again, it might attract attention from [Matt Painter](https://plugins.jetbrains.com/author/4e8b2560-41fe-4688-af5a-8116dbfccf2b) and [Paul](https://plugins.jetbrains.com/author/1dc9bf74-5e0d-49e6-a1cd-d4b2dcb15b50), which could potentially lead to some issues.
